lib/list.gd

```diff
@@ -869,6 +869,44 @@ DeclareGlobalFunction( "PositionSet" );
 DeclareOperation( "PositionProperty", [ IsList, IsFunction ] );
 DeclareOperation( "PositionProperty", [ IsList, IsFunction, IS_INT ] );
 
+#############################################################################
+##
+#O  PositionMaximum( <list> [, <func>] )
+#O  PositionMinimum( <list> [, <func>] )
+##
+##  <#GAPDoc Label="PositionMaximum">
+##  <ManSection>
+##  <Oper Name="PositionMaximum" Arg='list [, func]'/>
+##  <Oper Name="PositionMinimum" Arg='list [, func]'/>
```

Comment:
It does not invalidate GAPDoc, but since both `PositionMaximum` and `PositionMinimum` are functions, it would be more consistent to use `Func` instead of `Oper` in `<Oper Name="PositionMaximum" Arg='list [, func]'/>` (there is even an automated check for such things added in #538).